.github/workflows/release_library.yml

```diff
@@ -0,0 +1,31 @@
+name: Release Library
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version to release
+        required: true
+      force:
+        description: Force a release even when there are release-blockers (optional)
+        required: false
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: "Release a new librelay version"
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GH_RELEASE_PAT }}
+          fetch-depth: 0
+
+      - name: Prepare release
+        uses: getsentry/action-prepare-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_RELEASE_PAT }}
+        with:
+          version: ${{ github.event.inputs.version }}
+          force: ${{ github.event.inputs.force }}
+          path: py
```

Comment:
Why not parametrize the main release action instead?

Reply:
I feared that this would be too error prone. One is CalVer, the other one requires a version. One is released on schedule, the other only ever manually.

They do look quite similar, though.

Reply:
Hmm, we can set `CalVer` automatically and disable if path exists but I'm not sure if the complexity is worth it at this point. Maybe we can have the duplication until we have more then?

Reply:
Sounds good to me, let's go ahead with the current patch then. Thanks!